js/tests/unit/phantom.js

```diff
@@ -13,6 +13,8 @@
   QUnit.config.reorder = false
   // Run tests serially, not in parallel.
   QUnit.config.autorun = false
+  // Require assert.expect in each test.
```

Comment:
We want this enabled everywhere, not just for PhantomJS, so this should go in index.html instead.